src/regression.py

```diff
@@ -131,17 +131,21 @@ def random_forest_regression(train_x, train_y, output, train_data):
   random_rfr.fit(train_x, train_y)
   fold_accuracies_rfr = cross_val_score(estimator=random_rfr, X=train_x, y=train_y, cv=5)
   cv_scores = pd.DataFrame({'Fold': [1, 2, 3, 4, 5],
-                            'Neg Mean Squared Error': fold_accuracies_rfr})
+                            'Neg Mean Squared Error': np.around(fold_accuracies_rfr, 2)})
                     
   # Check that the regression worked and all cv_scores are not equal to zero
-  assert cv_scores['Neg Mean Squared Error'].all() != 0, "Random Forest Regression failed..."
+  assert cv_scores['Neg Mean Squared Error'].any() != 0, "Random Forest Regression failed..."
```

Comment:
What is the intention here? If just one of the scores is zero, should this fail? If so, I think you need `all() != 0` or `any() == 0`

Reply:
the `all() != 0` throws an error because if just one value is 0, it will stop the analysis (because they all HAVE to be not 0). The `any() != 0` throws a value only if all of them are zero (because as lone as one is not zero it won't throw an error. I had this backwards before, but didn't notice till it started throwing an error with the linear regression analysis.

Reply:
Ok, I though the intention was to indeed throw an error if just one was 0.